simple_message/include/simple_message/messages/robot_status_message.h

```diff
@@ -110,8 +110,38 @@ class RobotStatusMessage : public industrial::typed_message::TypedMessage
   {
     return this->status_.byteLength();
   }
+  
+  /**
+   * \brief Gets the communication type of the message
+   * 
+   * \return the value of the comm_type parameter (refer to simple_message::CommTypes::CommType)
+   */
+  int getCommType() const

```

Comment:
Is there a reason you added this method here instead of in the base class [TypedMessage](https://github.com/ros-industrial/industrial_core/blob/indigo-devel/simple_message/include/simple_message/typed_message.h)?  `TypedMessage` implements [getMessageType](https://github.com/ros-industrial/industrial_core/blob/indigo-devel/simple_message/include/simple_message/typed_message.h#L137) here.